src/components/Demonstration/TreeNetworkGraph.jsx

```diff
@@ -55,10 +56,40 @@ function TreeNetworkGraph({ data, width, height, onNodeClick = () => {} }) {
   const depth = Math.max(...data.nodes.map(e => e.level))
   const executionId = useMemo(() => data?.nodes[0]?.executionId, [data])
   const query = recentObservationsQuery(executionId)
-  const { data: observations } = useQuery(query.definition, query.options)
+  const { data: rawObservations } = useQuery(query.definition, query.options)
```

Comment:
You probably want to use `useQueryAll` instead, or you will be restricted to the 100 first observations